grpc.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'googleauth',      '~> 0.5.1'
 
   s.add_development_dependency 'bundler',            '~> 1.9'
+  s.add_development_dependency 'facter',             '~> 2.4'
```

Comment:
I need a way to count cores for the CoreCount RPC. This dependence should have already been there since #6010 , actually, but I had forgotten to include it.